src/error.rs

```diff
@@ -106,6 +130,12 @@ impl fmt::Display for Error {
 
             #[cfg(feature = "native-mux")]
             Error::InvalidCommand => write!(f, "invalid command: Command contains null byte."),
+
+            #[cfg(feature = "sftp")]
+            Error::SftpError(_) => write!(f, "Sftp error"),
```

Comment:
Well, if you look at format string for other variants in `fmt::Display`, you will find that the underlying error is not displayed, that's why I followed the suit here.

@jonhoo What's your thought here?

Should we change `fmt::Display` to also display error messages from the underlying errors?

Reply:
The other ones I think may be only instantiated in very specific cases where the message is specific enough that it's already understandable (not sure but that seems possible, which is why I didn't comment on them).
For this one specifically I'm pretty sure that "Sftp error" is not specific enough for the variety of errors that will possibly be contained in there.

Reply:
> For this one specifically I'm pretty sure that "Sftp error" is not specific enough for the variety of errors that will possibly be contained in there.

That is definitely true for for [`Error::SftpError`] since it has many variants and I suppose it also is true for [`Error::SshMux`].

[`Error::SshMux`]: https://docs.rs/openssh-mux-client/latest/openssh_mux_client/enum.Error.html
[`Error::SftpError`]: https://docs.rs/openssh-sftp-client/latest/openssh_sftp_client/enum.Error.html

Reply:
I have adopted your suggestion for `Error::SftpError`.

For `Error::SshMux`, I will do it in a separate PR.